libres/lib/config/conf.cpp

```diff
@@ -1077,6 +1088,26 @@ bool conf_instance_get_path_error(const conf_instance_type *conf_instance) {
     return path_errors;
 }
 
+/**
+ * This method is called from C. It uses the internal method above and
```

Comment:
"above" might get outdated quickly, and likewise the implementation detail about what functions it calls. I think a description about what sort of string to expect would be more helpful.